lib/galaxy/webapps/galaxy/api/tours.py

```diff
@@ -3,16 +3,52 @@
 """
 import logging
 
+from fastapi import Depends
+from fastapi_utils.cbv import cbv
+from fastapi_utils.inferring_router import InferringRouter as APIRouter
+
+from galaxy.app import UniverseApplication
+from galaxy.schema import (
+    TourDetails,
+    TourList,
+)
 from galaxy.web import (
     expose_api_anonymous_and_sessionless,
     legacy_expose_api,
     require_admin
 )
 from galaxy.webapps.base.controller import BaseAPIController
+from . import (
+    get_admin_user,
+    get_app,
+)
 
 log = logging.getLogger(__name__)
 
 
+router = APIRouter(tags=['tours'])
+
+
+@cbv(router)
+class FastAPITours:
+    app: UniverseApplication = Depends(get_app)
```

Comment:
Maybe something more specific here like:
```suggestion
    tour_registry: Registry = Depends(get_tour_registry)
```

where `get_tours_registry` is:

`````python
def get_tour_registry(app: UniverseApplication = Depends(get_app)) -> Registry:
    return app.tour_registry
````

Reply:
Done.